### DISCUSSION TITLE
Refresh button

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#477] [Ideas]
Currently to refresh the database I am opening a different page and then going back to the database. 
I am not sure if there is a quicker/easier way but a refresh button would be really handy.

**Comment by imed-ghomari** [comment]
Instead of trying to refresh the database manually, explain the use case where the database didn't refresh automatically and file it as a bug request. The database should refresh automatically when you need it (sort, convert property...).

Otherwise, you can check the docs https://rafaelgb.github.io/obsidian-db-folder/features/Viewing%20data/#refresh-database

**Reply by DannyHatcher-Git** [reply]
ah ok that makes sense. Will get to that 😁
